pulse/broker/src/main/java/gov/ca/emsa/pulse/service/AlternateCareFacilityService.java

```diff
@@ -60,6 +67,7 @@ public AlternateCareFacility getById(@PathVariable("acfId") Long acfId) {
 
 	@ApiOperation(value = "Create a new ACF")
 	@RequestMapping(value = "/create", method = RequestMethod.POST)
+	@Secured({"ROLE_ADMIN", "ROLE_ORG_ADMIN"})
```

Comment:
ROLE_PROVIDER needs to be able to create ACFs as well. If they're the first one to log in with a given ACF they'll automatically create it